### PR TITLE
Remove more outdated information

### PR DESCRIPTION
**Pull request by Margen67**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ permalink: /about/
 
 * Windows 8 or later
 * Intel Sandy Bridge or Haswell processor (supporting AVX or AVX2)
-* A Vulkan compatible GPU (NVIDIA preferred)
+* A Vulkan (or D3D12) compatible GPU
```
Comment (gibbed) [inline about.md]: Don't need the () here, just "Vulkan or D3D12 compatible GPU" should be fine.

File: faq.md [hunk 2]
```diff
@@ -12,18 +12,13 @@ permalink: /faq/
...
-emulator from *any* source, especially not YouTube videos and shady websites.
-Come on people. Jeez.
-
 ## Xenia crashes when I run it!
 
+Make sure your PC meets the system requirements of xenia.
 This project is in early development phases and is not meant to be used by those
 who aren't comfortable debugging the code. Issues filed with 'it's broken' are
 generally unhelpful, and in most cases the team knows what's broken (the
-assumption is everything). If it doesn't work for you and you cannot figure out
```
Comment (gibbed) [inline faq.md]: I don't think this needed to be removed?

File: faq.md [hunk 3]
```diff
@@ -12,18 +12,13 @@ permalink: /faq/
...
 Yes, <a href="download/">choose a download</a>.
 
-Don't be an idiot and download a binary claiming to be of this project. In fact,
-don't be an idiot and download *any* binary claiming to be an Xbox 360 or PS3
-emulator from *any* source, especially not YouTube videos and shady websites.
-Come on people. Jeez.
-
 ## Xenia crashes when I run it!
 
+Make sure your PC meets the system requirements of xenia.
```
Comment (gibbed) [inline faq.md]: Xenia should be capitalized, and "system requirements" could be a link to the section in the FAQ.

File: faq.md [hunk 4]
```diff
@@ -33,27 +28,27 @@ No. Do not post links or downloads to such resources or you will be banned.
 
 * Windows 8 or later
 * 64-bit x86 processor supporting AVX
-* A Vulkan compatible GPU from [this list](http://vulkan.gpuinfo.org/) (NVIDIA recommended)
+* A Vulkan (or D3D12) compatible GPU from [this list](https://vulkan.gpuinfo.org/)
```
Comment (gibbed) [inline faq.md]: Probably better to do something like:

A compatible GPU
* Vulkan supporting GPU from this list.
* D3D12 supporting GPU.

File: faq.md [hunk 5]
```diff
@@ -33,27 +28,27 @@ No. Do not post links or downloads to such resources or you will be banned.
...
 ### What kind of GPU do I need?
 
-Vulkan support and drivers are required. You can check if your GPU is supported by referencing [this list](http://vulkan.gpuinfo.org/).
+Vulkan support and drivers are required. You can check if your GPU is supported by referencing [this list](https://vulkan.gpuinfo.org/).
 
 To get full speed and compatibility the project adapted to Vulkan.
 OpenGL is no longer supported in this project and deprecated.
 
-### Can you add support for Windows Me? How about my Pentium II?
+### Can you add support for Windows 7? How about my Pentium II?
```
Comment (gibbed) [inline faq.md]: Unnecessary to be changed.

File: faq.md [hunk 6]
```diff
@@ -33,27 +28,27 @@ No. Do not post links or downloads to such resources or you will be banned.
...
 
 Support for older operating systems and processors will not be added unless a
 contributor steps up to build and maintain them. All active contributors are on
 modern systems and busy with higher priority tasks. Whether the emulator runs on
-Windows Vista or not doesn't matter if it can't run games.
+Windows 7 or not doesn't matter if it can't run games.
```
Comment (gibbed) [inline faq.md]: Unnecessary to be changed.

File: faq.md (hunk 5 above)
Comment (Margen67) [reply faq.md]: Windows 7 seems more relevant since people ask about it.

**Comment by Margen67** [comment]
The gallery is outdated and not really needed now that we have the subreddit and Discord server.
Should it be removed? @gibbed

File: faq.md (hunk 2 above)
Comment (gibbed) [reply faq.md]: I think that'd be fine, but changing "support" to something more like "community-provided support" might be a good idea.

File: faq.md (hunk 5 above)
Comment (gibbed) [reply faq.md]: The question is tongue-in-cheek, it's meant to show the audacity of the question (and Win7 isn't completely out of the question-- those are).

**Pull request merged**